ui/app/components/dropdowns/account-options-dropdown.js

```diff
@@ -14,12 +14,13 @@ module.exports = AccountOptionsDropdown
 // TODO: hook up to state, connect to redux to clean up API
 // TODO: selectedAddress is not defined... should we use selected?
 AccountOptionsDropdown.prototype.render = function () {
-  const { selected, network, identities, style, dropdownWrapperStyle, menuItemStyles } = this.props
+  // const { selected } = this.props
+  const { network, identities, style, dropdownWrapperStyle, menuItemStyles } = this.props
 
   return h(AccountDropdowns, {
     enableAccountOptions: true,
     enableAccountsSelector: false,
-    selected: selectedAddress,
+    // selected: selectedAddress,
```

Comment:
What is this change? It seems to be more than linting fixes

Reply:
`selectedAddress` is not defined. So this fixes the no-undef lint error without changing the values of `selected` property as passed to in `AccountDropdowns`